docs/fortranstyle.rst

```diff
@@ -325,3 +325,42 @@ This is to be preferred to the use of additional logical variables if possible.
 Part of the reason for this choice is that from Fortran 2008 onwards, optional
 arguments to subroutines and functions are treated as not-present if not
 allocated.
+
+
+File I/O
+========
+
+All files must be opened (connected to a descriptor) by the ``openFile()``
+routine, which initializes a ``type(TFileDescriptor)`` instance. Whenever
+possible, use the ``mode`` argument to specify the file opening type::
+
+  call openFile(fd, "test.dat", mode="r")
+
+The ``mode`` specifier has following possible values:
+
+  * ``r``: read (file must exist, it will be positioned at its start),
+
+  * ``r+``: read/write (file must exist, it will be positioned at its start),
```

Comment:
```suggestion
  * ``r+``: read/write (file must exist, the descriptor the start of the file contents),
```

Reply:
Needs an explanation of what the difference between r and r+ is.

Reply:
`r` is read, `r+` is read/write as stated above. Do you think, it means more explanation?